### PR TITLE
RF | Platform Functions > refactoring of tests (POM with generics)

### PR DESCRIPTION
**Pull request by AlexSharon**
Refactoring https://trello.com/c/5Xyl2q3z

### CODE REVIEW

File: src/test/java/model/platformFunctions/PlatformFunctionsBinPage.java [hunk 1]
```diff
@@ -58,13 +58,8 @@ public void assertRestoreMessage() {
...
 
-    public HomePage assertMessage(String expectedMessage) {
-        Assert.assertEquals(deleteMessage.getText(), expectedMessage);
-        return new HomePage(getDriver());
-    }
-
-    public HomePage assertRecycleBinIsEmpty() {
-        assertMessage("Good job with housekeeping! Recycle bin is currently empty!");
-        return new HomePage(getDriver());
+    public PlatformFunctions2Page assertRecycleBinIsEmpty() {
```
Comment (pavel-sipaty) [inline src/test/java/model/platformFunctions/PlatformFunctionsBinPage.java:61]: Asserts should not be in model classes.

**Review by pavel-sipaty — CHANGES_REQUESTED** [review]
Sergey said that Asserts should not be in model classes.

File: src/test/java/model/platformFunctions/PlatformFunctionsBinPage.java (hunk 1 above)
Comment (AlexSharon) [reply src/test/java/model/platformFunctions/PlatformFunctionsBinPage.java:61]: Please refer to the latest lesson of Sergei, time code 1.35:10

**Comment by AlexSharon** [comment]
> Sergey said that Asserts should not be in model classes.

Then he said different thing last week.

**Review by IrinaRiz — APPROVED** [review]

**Review by VasiliyTsoy — APPROVED** [review]

**Review by pavel-sipaty — APPROVED** [review]

**Pull request merged**